src/server/protocol.ts

```diff
@@ -904,6 +908,18 @@ namespace ts.server.protocol {
         end: Location;
     }
 
+    export interface TextRange {

```

Comment:
I'd be pretty surprised to learn we don't already have a type for this.  Also, that type (or types) presumably accounts for the fact that VS Code uses line & column, rather than offsets.

Reply:
Removed in favor of `FileRangeRequestArgs`.